index.html

```diff
@@ -125,6 +125,14 @@ <h2 property="schema:name">Work Items</h2>
                 </tbody>
 
                 <tbody>
+                  <tr>
+                    <td><a href="https://github.com/solid/webid-profile/blob/main/notes/pre-final-draft.md" rel="cito:citesForInformation">WebID Profile</a></td>
+                    <td><a href="https://github.com/solid/webid-profile">https://github.com/solid/webid-profile</a></td>
+                    <td>Unofficial Draft</td>
+                    <td>Ecosystem Proposal</td> 
+                    <td>Community Report</td>
+                    <td>TBD</td>
```

Comment:
```suggestion
                    <td>2022-06-30</td>
```

I suggest noting the expected completion date for a work item like this. It shows commitment and signal for potential implementers and contributors to the technical report. If the group needs to continue working on it, a request can be made to change the date.